### DISCUSSION TITLE
Unable to install Changedetection

### DISCUSSION THREAD

**Discussion by MrVue1217** [tteck/Proxmox#3502] [Help Needed] [answered]
Unable to install changedetection as receiving the error message below when cloning browserless. It get stuck here and doesnt proceed any further. Any suggestion. TIA

npm warn deprecated vm2@3.9.19: The library contains critical security issues and should not be used for production! The maintenance of the project has been discontinued. Consider migrating your code to isolated-vm.
npm warn deprecated intl-messageformat-parser@1.8.1: We've written a new parser that's 6x faster and is backwards compatible. Please use @formatjs/icu-messageformat-parser

![Screenshot 2024-08-09 at 3 22 51 PM](https://github.com/user-attachments/assets/39a95b2d-2ed2-45a1-98ca-24e514e0777c)

**Comment by test-perso** [comment]
I'm experiencing the same issue when I tried installing it today. It's stuck at "installing browserless & Playwright"
![Untitled](https://github.com/user-attachments/assets/aad63117-0a89-4016-9fd6-232dec2a4976)

**Comment by mohammedmulazada** [comment, marked as answer]
Should be fixed now with commit 72dfc20090ada0e197e3f681e95b4c839c69814e (thanks @tteck). Just tested myself.
